fvm/src/state_tree.rs

```diff
@@ -543,13 +541,13 @@ impl ActorState {
                 syscall_error!(InsufficientFunds; "when deducting funds ({}) from balance ({})", amt, self.balance).into(),
             );
         }
-        self.balance -= amt;
+        self.balance = &self.balance - amt;
```

Comment:
E.g., this is a case where we now have to allocate a new "balance". `self.balance -= amt` wouldn't need that.